### PR TITLE
definition of wetland emissions

### PR DESCRIPTION
**Pull request by flohump**
Updated definition of wetland emissions

### CODE REVIEW

File: definitions/variable/emissions/emissions.yaml [hunk 1]
```diff
@@ -113,10 +113,9 @@
       - Emissions|{Level-3 Species}|AFOLU|Land|Harvested Wood Products
       - Emissions|{Level-3 Species}|AFOLU|Land|Other
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions from land that is covered or saturated by water for
-      all or part of the year (e.g., peatland) and that does not fall
-      into the forest land, cropland, grassland or settlements
-      categories (partially IPCC 1996 category 4D/5A/5B/5E; IPCC 2006 category 3B4)
+    description: Net emissions from managed wetlands (e.g., peatland), including 
+      emissions from drained wetlands used for agriculture or forestry (IPCC 2006 categories 3B1, 3B2 and 3B3),
```
Comment (merfort) [inline definitions/variable/emissions/emissions.yaml:117]: These are the precise categories, right?
```suggestion
      emissions from drained wetlands used for agriculture or forestry (IPCC 2006 categories 3B1biii, 3B2biii and 3B3biii), 
```

Just for consistency, what about the other land types converted from wetlands, i.e. 
`3B5biv Wetlands Converted to Settlements`, `3B6biv Wetlands Converted to Other Land`?

Comment (flohump) [reply definitions/variable/emissions/emissions.yaml]: 3B5biv Wetlands Converted to Settlements and 3B6biv Wetlands Converted to Other Land is now included

**Review by merfort — COMMENTED** [review]
Looks good, thanks!

File: definitions/variable/emissions/emissions.yaml [hunk 2]
```diff
@@ -113,10 +113,9 @@
       - Emissions|{Level-3 Species}|AFOLU|Land|Harvested Wood Products
       - Emissions|{Level-3 Species}|AFOLU|Land|Other
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions from land that is covered or saturated by water for
-      all or part of the year (e.g., peatland) and that does not fall
-      into the forest land, cropland, grassland or settlements
-      categories (partially IPCC 1996 category 4D/5A/5B/5E; IPCC 2006 category 3B4)
+    description: Net emissions from managed wetlands (e.g., peatland), including
```
Comment (danielhuppmann) [inline definitions/variable/emissions/emissions.yaml:116]:
```suggestion
    description: Net emissions of {Level-3 Species} from managed wetlands (e.g., peatland),
```

File: definitions/variable/emissions/emissions.yaml [hunk 3]
```diff
@@ -113,10 +113,9 @@
       - Emissions|{Level-3 Species}|AFOLU|Land|Harvested Wood Products
       - Emissions|{Level-3 Species}|AFOLU|Land|Other
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions from land that is covered or saturated by water for
-      all or part of the year (e.g., peatland) and that does not fall
-      into the forest land, cropland, grassland or settlements
-      categories (partially IPCC 1996 category 4D/5A/5B/5E; IPCC 2006 category 3B4)
+    description: Net emissions from managed wetlands (e.g., peatland), including 
+      emissions from drained wetlands used for agriculture, forestry or other uses (IPCC 2006 categories 3B1biii, 3B2biii and 3B3biii, 3B5biv, 3B6biv),
```
Comment (danielhuppmann) [inline definitions/variable/emissions/emissions.yaml:117]:
```suggestion
      including emissions from drained wetlands used for agriculture, forestry or other uses
      (IPCC 2006 categories 3B1biii, 3B2biii and 3B3biii, 3B5biv, 3B6biv), 
```

File: definitions/variable/emissions/emissions.yaml [hunk 4]
```diff
@@ -113,10 +113,9 @@
       - Emissions|{Level-3 Species}|AFOLU|Land|Harvested Wood Products
       - Emissions|{Level-3 Species}|AFOLU|Land|Other
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions from land that is covered or saturated by water for
-      all or part of the year (e.g., peatland) and that does not fall
-      into the forest land, cropland, grassland or settlements
-      categories (partially IPCC 1996 category 4D/5A/5B/5E; IPCC 2006 category 3B4)
+    description: Net emissions from managed wetlands (e.g., peatland), including 
+      emissions from drained wetlands used for agriculture, forestry or other uses (IPCC 2006 categories 3B1biii, 3B2biii and 3B3biii, 3B5biv, 3B6biv), 
+      and emissions from rewetted wetlands (IPCC 2006 category 3B4).
```
Comment (danielhuppmann) [inline definitions/variable/emissions/emissions.yaml:118]: No full stop at the end of descriptions for consistency.
```suggestion
      and emissions from rewetted wetlands (IPCC 2006 category 3B4)
```

**Review by danielhuppmann — COMMENTED** [review]
I'd still keep the actual emission species in the description for consistency, see a suggestion inline.

Not sure how I feel about that this is the only variable where "net" is explicitly mentioned in the description...

**Comment by flohump** [comment]
> I'd still keep the actual emission species in the description for consistency, see a suggestion inline.
> 
> Not sure how I feel about that this is the only variable where "net" is explicitly mentioned in the description...

Thanks for the review and feedback.
I see your point on "net". "net" is correct but for consistency I removed it.

**Review by merfort — APPROVED** [review]
Thanks, looks good from my side

**Review by jkikstra — CHANGES_REQUESTED** [review]
Okay @flohump so if I understand this correctly, you want to quite clearly deviate from the [IPCC 2006 reporting categories](https://www.ipcc-nggip.iges.or.jp/public/2019rf/pdf/1_Volume1/19R_V1_Ch08_Reporting_Guidance.pdf).
Meaning - 'all emissions related to wetland change: both _converted to_ and _converted from_'.

Correct?

If so, then would be good to also update the one other Land category that will be affected by this change:
`Emissions|{Level-3 Species}|AFOLU|Land|Other Land Use and Land-Use Change`
... with something like '.... except under Land|Wetlands'

**Comment by flohump** [comment]
> Okay @flohump so if I understand this correctly, you want to quite clearly deviate from the [IPCC 2006 reporting categories](https://www.ipcc-nggip.iges.or.jp/public/2019rf/pdf/1_Volume1/19R_V1_Ch08_Reporting_Guidance.pdf). Meaning - 'all emissions related to wetland change: both _converted to_ and _converted from_'.
> 
> Correct?
> 
> If so, then would be good to also update the one other Land category that will be affected by this change: `Emissions|{Level-3 Species}|AFOLU|Land|Other Land Use and Land-Use Change` ... with something like '.... except under Land|Wetlands'

No. The intention is to revise the definition such that it covers `both _converted to_ and _converted from_`. 
The current definition only covers rewetted peatlands and excludes drained peatlands. But drained peatlands are by far the major source of GHG emissions from managed peatlands.

**Review by danielhuppmann — APPROVED** [review]
Looks good, thanks!

**Comment by jkikstra** [comment]
> _converted to_ and _converted from_

@flohump sorry to come back at this; but just to clarify. In my reading we both said the same?

The new variable contains "both converted to and converted from".

In IPCC categories, we have a category for each land type which includes:
- YES: stay the same land category (3 B * a)
- YES: converted to this land category (3 B * b)
- NO: converted from this land category to another category (this falls under the (3 B * b) of a different land category)

So if you have, as you wrote in the description, also `3B1biii, 3B2biii and 3B3biii, 3B5biv, 3B6biv` then could this create issues in the future with other variables? 
For now, it could at minimum be good to do a minimal change in the description of `Emissions|{Level-3 Species}|AFOLU|Land|Other Land Use and Land-Use Change` to 
"Emissions and removals from forest land, cropland, grassland, settlements, and other land that cannot be accommodated in other categories (partially IPCC 1996 category 5; IPCC 2006 category 3B except **3B1biii, 3B2biii, 3B3biii, 3B5biv, 3B6biv and** 3B4)."

**Comment by flohump** [comment]
@jkikstra 
Thanks! I now understand your point. I opened PR #236 to fix this.

**Pull request merged**